models/Asset/Document/ImageThumbnail.php

```diff
@@ -40,7 +43,7 @@ final class ImageThumbnail
     protected $page = 1;
 
     /**
-     * @param Model\Asset\Document $asset
+     * @param Model\Asset\Document|null $asset
```

Comment:
why asset is nullable here?

Reply:
See for example https://github.com/pimcore/pimcore/blob/89ecd1b81d4e955164027ca3a9129fa4b0d82679/models/Asset/Document.php#L97